audiomentations/augmentations/air_absorption.py

```diff
@@ -0,0 +1,171 @@
+    taken jhoyla's answer here:
+    https://stackoverflow.com/questions/14267555/find-the-smallest-power-of-2-greater-than-or-equal-to-n-in-python
+    """
+    return 1 if x == 0 else 2 ** (x - 1).bit_length()
+
+
+class AirAbsorption(BaseWaveformTransform):
+    """
+    Applies a wind absorption transform parametrized by temperature, humidity, and distance
+    of source.
+
+    This is not a scientifically accurate transform but basically applies a uniform
+    filterbank with attenuations given by:
+
+    att = exp(- distance * absorption_coefficient)
+
+    where distance is the microphone-source assumed distance in meters and `absorption_coefficient`
+    is adapted from a lookup table by pyroomacoustics [1]. It can also be seen as a lowpass filter
+    with variable octave attenuation.
+
+    Note: This only "simulates" the dampening of high frequencies, and does not  attenuate according to the distance law. Gain augmentation needs
+    to be done separately.
+
+    [1] https://github.com/LCAV/pyroomacoustics
+    """
+
+    supports_multichannel = True
+
+    # Table of air absorption coefficients adapted from `pyroomacoustics`.
+    # The last two columns have been extrapolated with `curve_fit`
+    air_absorption_table = {
+        "10C_30-50%": [
+            x * 1e-3 for x in [0.1, 0.2, 0.5, 1.1, 2.7, 9.4, 29.0, 91.5, 289.0]
+        ],
+        "10C_50-70%": [
+            x * 1e-3 for x in [0.1, 0.2, 0.5, 0.8, 1.8, 5.9, 21.1, 76.6, 280.2]
+        ],
+        "10C_70-90%": [
+            x * 1e-3 for x in [0.1, 0.2, 0.5, 0.7, 1.4, 4.4, 15.8, 58.0, 214.9]
+        ],
+        "20C_30-50%": [
+            x * 1e-3 for x in [0.1, 0.3, 0.6, 1.0, 1.9, 5.8, 20.3, 72.3, 259.9]
+        ],
+        "20C_50-70%": [
+            x * 1e-3 for x in [0.1, 0.3, 0.6, 1.0, 1.7, 4.1, 13.5, 44.4, 148.7]
+        ],
+        "20C_70-90%": [
+            x * 1e-3 for x in [0.1, 0.3, 0.6, 1.1, 1.7, 3.5, 10.6, 31.2, 93.8]
+        ],
+        "center_freqs": [125, 250, 500, 1000, 2000, 4000, 8000, 16000, 32000],
+    }
+
+    def __init__(
+        self,
+        min_temperature: float = 10.0,
+        max_temperature: float = 20.0,
+        min_humidity: float = 30.0,
+        max_humidity: float = 90.0,
+        min_distance: float = 10.0,
+        max_distance: float = 100.0,
+        p=0.5,
+    ):
+        """
+        :param min_temperature: Minimum temperature in celcius
+        :param max_temperature: Maximum temperature in celsius
+        :param min_humidity: Minimum humidity in percent %
+        :param max_humidity: Maximum humidity in percent %
+        :param min_distance: Minimum microphone-source distance in meters.
+        :param max_distance: Maximum microphone-source distance in meters.
+        :param p: The probability of applying this transform
+        """
+        assert float(min_temperature) in [
+            10.0,
+            20.0,
+        ], "Sorry, the only supported temperatures are either 10 or 20 degrees celcius"
```

Comment:
Why do you assert that the temperature can only take two values ? You could do as you already do for the humidity and the distance, ie directly map the temperature value it to the keys in the table. For example, if the user choose 12, you translate it to 10.

Reply:
> You could do as you already do for the humidity and the distance, ie directly map the temperature value it to the keys in the table. 

If a better model is added then that would be the case but here the keys have been chosen for distinct temperatures but ranges of humidity so I keep it like that to be used as originally intended.